timon/webclient/webif1/src/components/Timon.vue

```diff
@@ -231,27 +261,53 @@ export default {
         }
       }
     },
+    parse_host_group(host_group, fields=[], deepness=0){
+      let end_list = [];
+      for(let entry of Object.values(host_group)){
+          fields[deepness] = entry["name"];
+          if(entry["entries"] !== undefined && entry["entries"].length > 0){
+              if(typeof(entry["entries"][0])==="string"){
+                  for(let server of entry["entries"]){
+                      end_list.push({
+                          "name": server,
+                          "grp_change": [...fields]
+                      });
+                      for(let field in fields){
+                          fields[field] = null;
+                      }
+                  }
+              }
+              else{
+                  end_list = end_list.concat(this.parse_host_group(entry["entries"], fields, deepness+1));
+              }
+          }
+      }
+      return(end_list);
+    },
     parse_cfg (cfg) {
```

Comment:
might add a doc string. 
Somethign along the lines.
Parses configuration / determines order of hosts to be displayed, ...